packages/js/admin-layout/tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
 	"extends": "../tsconfig",
+	"include": [ "src/" ],
 	"compilerOptions": {
 		"rootDir": "src",
 		"outDir": "build-module",
 		"declaration": true,
 		"declarationMap": true,
 		"declarationDir": "./build-types",
-		"typeRoots": [
-			"./typings",
-			"./node_modules/@types"
-		]
+		"typeRoots": [ "./typings", "./node_modules/@types" ]
```

Comment:
Just noting quite a few packages mention typings as a type root but have no `typings` directory. It doesn't cause an issue for TS so I've opted to leave `typeRoots` alone.